client/src/pages/Dashboard.js

```diff
@@ -2,21 +2,35 @@ import React, { useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import { useDispatch, useSelector } from 'react-redux';
 import { getUserInfo } from '../redux/actions/userActions';
+import { getTwitterProfileInfoData } from '../redux/actions/twitterActions';
+import DisplayUserMessages from '../components/DisplayUserMessages';
+import ConnectTwitterButton from '../components/ConnectTwitterButton';
+import { isTwitterConnected } from '../services/isTwitterConnected';
 
 const Dashboard = () => {
   useSelector((state) => state.userReducer.present);
+  useSelector((state) => state.twitterReducer.isConnected);
   const dispatch = useDispatch();
 
   useEffect(() => {
     dispatch(getUserInfo());
   }, [dispatch]);
 
+  const getTwitterInfo = () => {
```

Comment:
Why use a wrapper function for calling a function?

Reply:
Good point. I got used to doing that I suppose. This is for the temporary button to get user's twitter info, so it will be deleted soon. I will keep in mind this suggestion

Reply:
Update: 
Removed wrapper function